src/winetricks

```diff
@@ -11557,16 +11557,16 @@ load_peverify()
 w_metadata physx dlls \
     title="PhysX" \
     publisher="Nvidia" \
-    year="2014" \
+    year="2019" \
     media="download" \
-    file1="PhysX-9.14.0702-SystemSoftware.msi" \
+    file1="PhysX-9.19.0218-SystemSoftware.exe" \
     installed_file1="${W_PROGRAMS_X86_WIN}/NVIDIA Corporation/PhysX/Engine/v2.8.3/PhysXCore.dll"
 
 load_physx()
 {
-    w_download https://uk.download.nvidia.com/Windows/9.14.0702/PhysX-9.14.0702-SystemSoftware.msi 0a022e28accf5851be9d6577487cdcd3d3a3e2a8a21a64456b72b415c217f03c
+    w_download https://uk.download.nvidia.com/Windows/9.19.0218/PhysX-9.19.0218-SystemSoftware.exe 4f36389fcbfbdef4781fb85e7a68373542903235f65d93f7143693738324917c
     w_try_cd "${W_CACHE}/${W_PACKAGE}"
-    w_try "${WINE}" msiexec /i PhysX-9.14.0702-SystemSoftware.msi ${W_OPT_UNATTENDED:+/q}
+    w_try "${WINE}" PhysX-9.19.0218-SystemSoftware.exe ${W_OPT_UNATTENDED:+/q}
```

Comment:
The newer physx doesn't support /q; looks like it uses /s now.

Reply:
Yeah, it seems so. Still it will not actually allow you to do a true silent install, cos it seems you still have to click "I agree" button on the license agreement.
I updated it and added a filecheck that i missed. `v2.8.3/PhysXCore.dll` exists on both, although a different hash (and i dunno if you can do a hash-check with `installed_file`?)
The folder and file `86C5F4F22ECD/APEX_Particles_x64.dll` does not exist on the 9.14 version tho, so it will allow an upgrade without failing with "already installed".

Sorry i missed that.

Reply:
It installs fine for me (i.e., without a popup for the license) with wine-6.12-195-gc586c57082d if I use -s.